openmetadata-ui/src/main/resources/ui/src/jsons/en.ts

```diff
@@ -164,6 +164,8 @@ const jsonData = {
     'unauthorized-user': 'UnAuthorized user! please check email or password',
     'fetch-re-index-all': 'Error while fetching re index data!',
     'update-re-index-all': 'Error while re indexing!',
+
+    'restore-table-error': 'Error while restoring table',
```

Comment:
Add error and success messages for every entity.

Reply:
@Ashish8689 Do interpolation to achieve that I advised earlier to create Error/Success message for entity